node/node.go

```diff
@@ -430,6 +459,10 @@ func (node *AlgorandFullNode) Stop() {
 	node.lowPriorityCryptoVerificationPool.Shutdown()
 	node.cryptoPool.Shutdown()
 	node.cancelCtx()
+	node.ledger.Close()
```

Comment:
Closing ledger is a gamble now, we have a [WIP task](https://github.com/algorand/go-algorand-internal/issues/3051) to fix this